### DISCUSSION TITLE
没有menu菜单吗

### DISCUSSION THREAD

**Discussion by OldBaby001** [hellof2e/quark-design#280] [Q&A]
没有menu菜单吗，什么时候可以推一个下拉菜单，普通菜单的

**Comment by sanqi-med** [comment]
> 没有menu菜单吗，什么时候可以推一个下拉菜单，普通菜单的

menu不是pc场景才用的吗？下拉可以直接基于popup包装哈，也欢迎提PR哈

**Reply by OldBaby001** [reply]
https://github.com/hellof2e/quark-doc-header
也就是在pc端使用的，quarkd是偏移动端使用，有开放封装pc端的ui库吗

**Comment by sanqi-med** [comment]
> https://github.com/hellof2e/quark-doc-header 也就是在pc端使用的，quarkd是偏移动端使用，有开放封装pc端的ui库吗

暂时没有 PC 端 UI 库哈，可以使用我们的 quarkc 构建 PC 组件库

**Reply by OldBaby001** [reply]
Got，感谢解答
